R/derive_vars_transposed.R

```diff
@@ -96,6 +112,21 @@
 #'     value_var = FASTRESC
 #'   ) %>%
 #'   select(USUBJID, CMDECOD, starts_with("CMATC"))
+#'
+#' # Note: the id_vars argument here is key to uniquely identify
+#' # rows of dataset_merge and avoid duplicates-related errors.
+#' # Compare the above call with:
```

Comment:
```suggestion
#' # Note: the `id_vars` argument here is needed to uniquely identify
#' # rows of dataset_merge and avoid duplicates-related errors.
#' # Compare the above call when `id_vars = NULL`:
```

Reply:
we say the word key a lot for variable keys so i got confused when seeing it here.